tests/ct_connection_vhost_ports_with_netperf_udp.py

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) 2022 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+DPDK Connection Tracking with vhost Port for Netperf
+"""
+
+# in-built module imports
+import time
+import sys
+
+# Unittest related imports
+import unittest
+
+# ptf related imports
+import ptf
+import ptf.dataplane as dataplane
+from ptf.base_tests import BaseTest
+from ptf.testutils import *
+from ptf import config
+
+# scapy related imports
+from scapy.packet import *
+from scapy.fields import *
+from scapy.all import *
+
```

Comment:
The line 34-37 scapy are never used. Only netperf traffic. Suggest to remove them for clean code.